src/mousetracker.js

```diff
@@ -2879,6 +2879,12 @@
                 }
             }
 
+            // A primary mouse button may have been released while the non-primary button was down
+            if (pointsList.contacts > 0 && pointsList.type === 'mouse') {
+                // Stop tracking the mouse
+                pointsList.contacts--;
+                return true;
```

Comment:
Okay, that all makes sense. Well, maybe we should land this as is and see how it goes; it's certainly the best solution I'm aware of. 

@LarissaSmith how about adding a "see https://github.com/openseadragon/openseadragon/pull/1223" comment there to point back to this conversation? Otherwise I think it's ready to land if you're happy with it.